services/ui-src/src/components/modals/DeleteDynamicFieldModal.tsx

```diff
@@ -0,0 +1,64 @@
+// components
+import { Text } from "@chakra-ui/react";
+import { Spinner } from "@cmsgov/design-system";
+import { Modal } from "components";
+import { useState } from "react";
```

Comment:
Info: Would usually import this first since it's a top-level react thing, but it's also nbd.